doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -25,15 +25,143 @@
+
+Containerised Infrastructure Manager (CIM): a functional block that is responsible for controlling and managing the containerised NFVI compute, storage and network resources. Note CIM supports the Life Cycle Management of conatinerised and the underlying physical resources.
+
+"Architecture" in this chapter refers to NFVI + VIM (as specified in Reference Model Chapter 3) and consists of the Physical resources, virtual and container resources, and Software Management Layer (Hypervisor, Container runtime, virtual or container Orcehstrator(s), Operating System).
 
 <a name="2.3.1"></a>
 ### 2.3.1 General
 
 | Ref # | sub-category | Description |
 |----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `req.gen.ost.01` | Open source | The Architecture **must** use Kubernetes APIs.|
-| `req.gen.ost.02` | Open source | The Architecture **must** support dynamic request and configuration of virtual resources (compute, network, storage) through Kubernetes APIs. |
+| `req.gen.k8s.01` | Open source | The Architecture **must** use Kubernetes APIs.|
+| `req.gen.k8s.02` | Open source | The Architecture **must** support dynamic request and configuration of resources (compute, network, storage) through Kubernetes APIs. |
+| `req.gen.cnt.01` | Cloud nativeness | The Architecture **should** consist of stateless service components. However, where state is required it must be kept external to the component. |
+| `req.gen.cnt.02` | Cloud nativeness | The Architecture **should** consist of service components implemented as microservices that are individually dynamically scalable. |
+| `req.gen.scl.01` | Scalability | The Architecture **should** support policy driven horizontal auto-scaling of workloads. |
+| `req.gen.rsl.01` | Resiliency | The Architecture **must** support resilient Kubernetes components that are required for the continued availability of running workloads. |
+| `req.gen.rsl.02` | Resiliency | The Architecture **should** support resilient Kubernetes service components that are not subject to `req.gen.rsl.01`. |
+| `req.gen.avl.01` | Availability | The Architecture **must** provide High Availability for Kubernetes components. |
+| `req.gen.ref.01` | Model | The Architecture **must** support the Reference Model defined profiles (instance types: Basic, Network Intensive, Compute Intensive). |
+
+
+<p align="center"><b>Table 2-1:</b> Containerised Platform: General Requirements.</p>
```

Comment:
Suggest the table title is changed to "Kubernetes Architecture: General Requirements"

Reply:
Thanks -- change made